### PR TITLE
feat: add a container response filter to append API-Version to all the response headers

### PR DESCRIPTION
**Pull request by kapil-agnihotri**
ING-3735

### CODE REVIEW

File: deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+	// Whether to add API version to the header or not
+	private boolean addApiVersionToHeader = true;
+
+	// Constant for API-Version
+	private static final String API_VERSION_CONSTANT = "API-Version";
+
+	// API version information based on
+	// https://github.com/opengeospatial/ogcapi-features#using-the-standard
+	private static final String API_VERSION = "1.0.1";
```
Comment (stempler) [inline deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java]: Please use the existing constant in `OpenApiCreator` instead, and change that to `1.0.1`.

File: deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package org.deegree.services.oaf.resource;
```
Comment (stempler) [inline deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java]: Please add to `org.deegree.services.oaf.filter` instead, then it is consistent with the other added filters.

File: deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java (hunk 1 above)
Comment (kapil-agnihotri) [reply deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java]: Done.

File: deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java (hunk 2 above)
Comment (kapil-agnihotri) [reply deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java]: Done.

**Review by stempler — APPROVED** [review]

**Pull request merged**